### PR TITLE
Updated installers

### PR DESCRIPTION
**Pull request by Goldorion**
This PR updates the installers to change MCreator's elements by MCToolkit's elements. However, I didn't change the icons because I don't know how to get the icons with these file types.
I tested the Windows installer and it works fine. If you are on MAC or Linux, please test it.

### CODE REVIEW

File: platform/linux/mcreator.sh [hunk 1]
```diff
@@ -19,8 +19,8 @@ Name=MCreator
 Icon=${pwd}/icon.png
 EOL
 
-chmod +x mcreator.desktop
-cp mcreator.desktop ~/Desktop/mcreator.desktop
+chmod +x mctoolkit.desktop
+cp mctoolkit.desktop ~/Desktop/mcreator.desktop
```
Comment (Max094Reikeb) [inline platform/linux/mcreator.sh:23]: Here, you forgot to change the second `mcreator`
`cp mctoolkit.desktop ~/Desktop/mctoolkit.desktop`

**Review by Max094Reikeb — CHANGES_REQUESTED** [review]
Check comment

File: platform/linux/mcreator.sh (hunk 1 above)
Comment (Goldorion) [reply platform/linux/mcreator.sh]: Oops, fixed

**Review by Max094Reikeb — APPROVED** [review]
Seems good!

**Comment by Goldorion** [comment]
> 
> 
> Seems good!

Did you test to install for Mac versions?

**Comment by Max094Reikeb** [comment]
> > Seems good!
> 
> Did you test to install for Mac versions?

No I did not

**Pull request merged**